packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenuControl.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/OverflowMenu/overflow-menu';
+
+export interface OverflowMenuControlProps extends React.HTMLProps<HTMLDivElement> {
+  /** Any elements that can be rendered in the menu */
+  children?: any;
+  /** Additional classes added to the OverflowMenuControl */
+  className?: string;
+  /** Triggers the overflow dropdown to persist at all viewport sizes */
+  additionalOptions?: boolean;
```

Comment:
not really clear on what this does, description makes it sound like it could be better named `alwaysPersist`?

Reply:
Updated naming for clarity - no longer matches naming for the CSS rule it triggers (alwaysPersist && styles.modifiers.additionalOptions) but I agree that a clearer name makes more sense.